### PR TITLE
feat(decision-table/head): display FEEL as default expression language

### PR DESCRIPTION
**Pull request by barmac**
__Which issue does this PR address?__

This popped up on Slack as part of general feedback. The PR makes sure that we display default expression language (FEEL) when none is set.

__Definition of Done__

* [x] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [x] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [x] passes Continuous Integration checks
* [x] available as feature branch on GitHub
  * [x] contains the cleaned up commit history
  * [x] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [x] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: packages/dmn-js-decision-table/test/spec/features/decision-table-head/editor/input/InputEditorCellSpec.js [hunk 1]
```diff
@@ -90,57 +90,14 @@ describe('decision-table-head/editor - input', function() {
   }));
 
 
-  describe('should transform to script', function() {
```
Comment (pinussilvestrus) [inline packages/dmn-js-decision-table/test/spec/features/decision-table-head/editor/input/InputEditorCellSpec.js:93]: Removing the script feature isn't part of this PR, right? Maybe it makes sense to split the commit in removing the test case and the original improvement aka setting the default expression language.

**Comment by pinussilvestrus** [comment]
Related to https://github.com/bpmn-io/dmn-js/issues/527?

File: packages/dmn-js-decision-table/test/spec/features/decision-table-head/editor/input/InputEditorCellSpec.js (hunk 1 above)
Comment (barmac) [reply packages/dmn-js-decision-table/test/spec/features/decision-table-head/editor/input/InputEditorCellSpec.js:93]: I will do that.

**Comment by barmac** [comment]
> Related to #527?

Indeed. I will link the issue.

File: packages/dmn-js-decision-table/test/spec/features/decision-table-head/editor/input/InputEditorCellSpec.js (hunk 1 above)
Comment (barmac) [reply packages/dmn-js-decision-table/test/spec/features/decision-table-head/editor/input/InputEditorCellSpec.js:93]: Done via https://github.com/bpmn-io/dmn-js/pull/537/commits/7380a36254eb3be8bac14935a293f0a1b84945c8

**Comment by barmac** [comment]
> Related to #527?

Linked in https://github.com/bpmn-io/dmn-js/pull/537/commits/7ff6c85eba4e9f5f9939961977222aacc67ec4f7

**Pull request merged**